src/pygeom.c

```diff
@@ -113,40 +168,40 @@ static PyObject *GeometryObject_str(GeometryObject *self)
     return GeometryObject_ToWKT(self, "%s");
 }
 
+static PyObject *GeometryObject_reduce(PyObject *self)
```

Comment:
A docstring / comment would be helpful here, as the function name alone does not define its purpose.